app/layout.tsx

```diff
@@ -43,6 +165,33 @@ export const metadata: Metadata = {
   },
 };
 
+const jsonLd = {
+  "@context": "https://schema.org",
+  "@type": "WebApplication",
+  name: "Noblocks",
+  description:
+    "The first interface for decentralized payments to any bank or mobile wallet",
+  applicationCategory: "FinanceApplication",
+  operatingSystem: "Web",
+  offers: {
+    "@type": "Offer",
+    price: "0",
+    priceCurrency: "USD",
+  },
+  featureList: [
+    "USDC payments",
+    "USDT payments",
+    "DAI payments",
+    "Instant stablecoin transfers",
+    "Low-fee crypto remittance",
+    "Cross-border stablecoin payments",
+    "Crypto to bank transfers",
+    "Crypto to mobile money",
+  ],
+  supportedNetworks: ["Polygon", "Base", "Arbitrum", "BNB Chain", "Ethereum"],
```

Comment:
Celo, Lisk, and Optimism should be here

Reply:
[9e7f631](https://github.com/paycrest/noblocks/commit/9e7f63192c17f991740768d135d801399dcc2ea6)